.github/workflows/build.yml

```diff
@@ -38,9 +38,9 @@ jobs:
             arch: "ppc64le"
           - os: ubuntu-20.04
             arch: "s390x"
-          - os: windows-2016
+          - os: windows-2019
```

Comment:
This is the one surprise, by the way. It works fine with `windows-2019`, but finds GCC with `windows-2016`. I don't think any of the other things we've tried have this problem, and we test on `windows-2016`, but it might be worth further investigation.